frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -112,6 +114,60 @@ function hasAnyWireframes(snapshotData: Record<string, any>[]): boolean {
     })
 }
 
+function isCompressedEvent(ev: unknown): ev is compressedEventWithTime {
+    return typeof ev === 'object' && ev !== null && 'cv' in ev
+}
+
+function unzip(compressedStr: string): any {
+    return JSON.parse(strFromU8(gunzipSync(strToU8(compressedStr, true))))
+}
+
+function decompressEvent(ev: eventWithTime | compressedEventWithTime): eventWithTime {
```

Comment:
this is slightly ugly but I assume it'll change as we discover things so avoiding factoring this until i know its shape